docs/developer-manual/deployments.md

```diff
@@ -0,0 +1,107 @@
+---
+id: orcfax deployments
+sidebar_position: 3
+slug: /deployments
+---
+
+# Protocol deployments
+
+Signposting current and historical deployments of the Orcfax protocol. The
+hashes can be used to navigate Orcfax fact-statements and archival packages
+on Cardano and Arweave blockchains.
+
+This document serves as a reference for historical access to Orcfax data.
+Integrators should consult the [integrator docs][consume] for an explanation
+an the role of different scripts used in the protocol and how to correctly use
+the Orcfax protocol.
+
+## V1
+
+Orcfax V1 introduces the FSP (fact statement pointer) script. The FSP will not
+change in the lifetime of a deployment. The FSP is used to record the current
+FS (fact statement) script hash. The FSP, FS, and C (constitution) script
+hashes are documented below for each V1 deployment.
+
+### V1: Mainnet
+
+A summary of mainnet deployments is listed below.
+
+| Status | TYPE | FSP | FS | Arweave Hash |
```

Comment:
> What is this TYPE ? (as shown under preview) The type of the feed is completely independent of the deployment

It will be CER -- and whether or not we include the price of gold or anything else in the same feed is still to be determined. Because we only have CER right now and the other decision is still to be made this doesn't feel super uncomfortable to include here.

Reply:
> That this is in a single table suggests that they are coupled. We could change the FS script without changing the Arweave hash or vice versa.

> The table rows that include both FSP and FS hash... suggest they are related as 'siblings' rather than 'parent-child'.

I largely understand, but we don't mention relationship anywhere -- these are just three things that co-exist. When we ask `show-refs` we also get this information in linear form, and that's all I'm attempting to do here. So, when the FS is upgraded, another row is created and it will have FSP, FS (changed) and C. And if there's a constitution shared across deployments, it will reflect that. 

I get that as the author of the protocol you do see these things in a relationship, but these docs are really trying to be matter of fact. 

The reason I am really pushing for a table is it gives us the tools to quickly make a new row and document the new information and stay clear away from formatting/narrative.

If we can find a more clever way to tabularize this, it would be ideal, but really just trying to be pragmatic here and document the things in front of us now. 

Also, I do think, this is not for integrators, once they read the consume docs, they will also get a better picture of the protocol like yourself. This is just a historical record/reference to help revisit the scripts in future.

Is there room to improve? maybe, do we have to be perfect now? we just have to be good enough.

Reply:
For now, sure. 
But in the same vein as the overall comment: 
it is _incidental_ that there is a single feed-type for the deployment. Incidental since we yet have no other feed-types. 
It is not _structural_. And the way this is presented suggests it it structural that there is a relationship of one feed-type per deployment.

Reply:
The reply above was to the previous comment, while the newer comment had not been received. 

> The reason I am really pushing for a table is it gives us the tools to quickly make a new row and document the new information and stay clear away from formatting/narrative.

The suggestion had a table:
https://github.com/orcfax/docs/blob/waalge/suggestion-for-policies-page/docs/developer-manual/policies.md#v10
It wasn't in any way explained. But it was a table of data of the list of updates of an FSP. 

I really think people will look at this and think:

- "Oh so there must be one deployment per feed type" which is currently technically true, but incidentally so. 
- "Wait why does it say 'mainnet deployment**S**', is there more than one?! wait why are there _different_ deployments with the same FSP hash?!"

 I'll let @Christian-MK come in with comments

Reply:
> The suggestion had a table:

It did and formed the basis of this PR -- but the problem for me is the other "data" in bullet points. It gets messy to maintain (it's where these pages started) and be consistent about, hence trying to consolidate different ideas.

Using the Tx hash in the table seemed unnecessary given what we know we can do with the FSP to get the previous hashes. I'd argue the most important dates are init (and the slot/blockhash) and teardown for completeness. But lots of this is subjective. 

If you can find a way to approach this where more of the data is structured I'd appreciate it. 

We have an mdx extension now, maybe we can use JSON?

Reply:
As a layperson, I dont suffer the confusion that @waalge is concerned may occur with a more technical reader. However, a technical reader will have gone through, or will shortly thereafter, the consume doc which explains why (and under what kinds of circumstances) a new deployment would occur which should address the confusion; perhaps it would be helpful to link to the doc or those sections?

Otherwise the tables seem clear and concise to me.

Reply:
```
### V1: Mainnet

A summary of mainnet deployments is listed below.
```
Would it make it clearer @waalge to change this to something like, `A summary of current and historical mainnet deployments are listed below.` ?
repeats language found in the opening paragraph

Reply:
Sort of case in point here 

> why ... a new deployment would occur which should address the confusion

A new deployment should _never_ occur outside testing. If you are lead to believe that we even contemplate multiple deployments might occur on mainnet, then something has gone wrong. 
The consume doc says nothing about a deployment, until the final section and that section contains only the FSP hash

Reply:
I think we're talking past eachother. Is it true that the FSP will **never** change/be replaced by another?

If yes, then `deployment` perhaps should refer to the FS. this way, the table may look something like this as business needs require us to update the FS:

```
| Status | TYPE | FSP | FS | Arweave Hash |
|--------|------|---- |----|--------------|
| TBD | CER | `1111` | `aaaa` | TBD |
| TBD | CER | `1111` | `bbbb` | TBD |
| TBD | CER | `1111` | `cccc` | TBD |
```

If no, and there are circumstances which would prompt us to change/replace an FSP, then whats the issue with referring to it as a deployment
```
| Status | TYPE | FSP | FS | Arweave Hash |
|--------|------|---- |----|--------------|
| TBD | CER | `1111` | `aaaa` | TBD |
| TBD | CER | `1111` | `bbbb` | TBD |
| TBD | CER | `2222` | `cccc` | TBD |
```

Reply:
Perhaps. Perhaps "deployment" now needs to be added to the glossary. There is no strict or precise definition 
of a deployment 
https://en.wikipedia.org/wiki/Software_deployment

IMO, the more relevant encapsulation is starting from the FSP. 
It is the FSP is what we promise is fixed. 
The FSP is the entry point to using Orcfax on-chain.
A consumer wishing to verify an integrator is using orcfax will be looking up the FSP hash. 

Consumers shouldn't need to notice if we upgrade the FS (including update the FSP state). 
They should be able to use their favourite dapp during an FS upgrade. 
Its moot, but given that the entry point does not move in an update, calling an FS upgrade a new "deployment" is odd.